src/Html.php

```diff
@@ -1204,7 +1204,11 @@ public static function includeHeader(
         // load log filters everywhere
         Html::requireJs('log_filters');
 
-        $tpl_vars['css_files'][] = ['path' => 'lib/tabler.css'];
+        if ($_SESSION['glpiisrtl'] ?? false) {
```

Comment:
Seems a bit strange rtl is not defined here, no? 
It would mean we try to display a page without any lang setup - I'm not sure it should happen. On which case was this variable missing?